crates/testing/src/slt/cli.rs

```diff
@@ -145,7 +145,8 @@ impl Cli {
         let mut tests: Vec<_> = if let Some(patterns) = &self.tests_pattern {
             let patterns = patterns
                 .iter()
-                .map(|p| glob::Pattern::new(p))
+                .map(|p| p.trim_end_matches(".slt"))
```

Comment:
What's this for? Is it to allow for things like `just slt "sqllogictests/*.slt"`?

Reply:
yeah, i use tab completion a lot, which always puts in the `.slt`. This previously failed, now it just ignores the extension if its there.